domains/transfer/Transfer.API/API/Cvr/CvrClient.cs

```diff
@@ -0,0 +1,40 @@
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading.Tasks;
+using API.Cvr.Models;
+
+namespace API.Cvr;
```

Comment:
Move this all files within the namespace to `API.CvrClient`

Reply:
Moved to Clients.Cvr. Else I would end up with CvrClient.CvrClient every time I needed the object